grails-app/controllers/au/org/ala/ecodata/HarvestController.groovy

```diff
@@ -152,13 +152,14 @@ class HarvestController {
                 String hostname = project.isMERIT ? grailsApplication.config.getProperty("fieldcapture.baseURL") : grailsApplication.config.getProperty("biocollect.baseURL")
                 DocumentHostInterceptor.documentHostUrlPrefix.set(hostname)
                 String filename = "darwin-core-${projectId}.zip"
+                boolean force = params.force ? params.force.toBoolean() : false
```

Comment:
params.getBoolean('force', false) is probably the more idiomatic way to do this?